src/components/ContactUs/ContactUs.tsx

```diff
@@ -0,0 +1,25 @@
+import { ReactElement } from 'react';
+
+import { FullLogoImage } from '../FullLogoImage';
+
+export interface ContactUsProps {
```

Comment:
you can remove `export` because this type is not used anywhere in the code. Set `export` only when you plan to use this type in some other files